### PR TITLE
integrate travis ci

### PR DESCRIPTION
**Pull request by fschr**

### CODE REVIEW

**Comment by pranaygp** [comment]
This shouldn't be go specific. We want the test file to run anything, not just go code

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "language": "go",
```
Comment (pranaygp) [inline .travis.yml]: This shouldn't be go. I think it should be bash. We're running a bash script for Travis tests.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  "go": "1.7.x",
+  "script": "./test",
```
Comment (pranaygp) [inline .travis.yml]: rename test script to test.sh

File: .travis.yml (hunk 1 above)
Comment (fschr) [inline .travis.yml]: go makes it install Go 1.7 and run `go get github.com/fschr/go`. If this were bash it wouldn't do that, I think.

Comment (fschr) [inline .travis.yml]: also, see https://github.com/travis-ci/travis-ci/issues/4090

Comment (pranaygp) [inline .travis.yml]: Ah. That issue is interesting. Definitely something travis needs to fix

**Pull request merged**